command-line-local-shortener/templates/index.html

```diff
@@ -1,21 +1,31 @@
-<!doctype html>
-<html>
-
-<head>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Golang URL shortener</title>
-</head>
-
-<body>
-    <p>example paragraph<strong>example</strong> example
-        <strong>example</strong>example</p>
+    <link rel="stylesheet" href="style.css" />
+  </head>
 
+  <body>
+    <!-- <p>example paragraph<strong>example</strong> example
+        <strong>example</strong>example</p>  -->
+    <!-- maybe do an onclick instead of making a whole new form??  -->
```

Comment:
Nah, this is still a standard use. It just sends the data as `application/x-www-form-urlencoded` or `multipart/form-data`.